### ISSUE TITLE
unexpected doc generated for custom array `body` param

### ISSUE THREAD

**Issue by wuhuizuo** [swaggo/swag#550]
**issue description**
unexpected doc generated for custom array `body` param

**Your swag version**
e.g. 1.6.3

**Your go version**
e.g. 1.13.1

golang doc for router handler method: (example @Param):

```golang
// @Param timelines body []db.Timeline true "timeline list"
```

json generated:
```json
                "parameters": [
                    {
                        "description": "timeline list",
                        "name": "timelines",
                        "in": "body",
                        "required": true,
                        "schema": {
                            "type": "db.Timeline", // not expect
                            "items": {
                                "$ref": "#/definitions/db.Timeline"
                            }
                        }
                    }
                     ...
```

doc Web look not correct:
![image](https://user-images.githubusercontent.com/2574558/68186124-4eb7c600-ffde-11e9-9598-a07555bbe9fd.png)

but should output: 
```json
                "parameters": [
                    {
                        "description": "timeline信息列表",
                        "name": "timelines",
                        "in": "body",
                        "required": true,
                        "schema": {
                            "type": "array",  // expect
                            "items": {
                                "$ref": "#/definitions/db.Timeline"
                            }
                        }
                    }
                    ...
```

**Comment by gmarcial** [comment]
Ya

I find myself in the same problem...

**Comment by trungtin719tt** [comment]
I got the same issue. Please tell me if anyone have the solution for this. Thank you.

**Comment by sdghchj** [comment]
#579

**Comment by easonlin404** [comment]
fixed by #579  please reopen it if there are any problems.
